docs/admin-guide/zope-manager-users.md

```diff
@@ -107,3 +107,45 @@ When you run the script, if the user already exists:
     Maybe the user already exists and nothing is done then.
     Or the implementation does not give info when it succeeds.
     ```
+### `addzopeuser` script in dockerized setup
+
+#### Prerequisites
+-   A running docker swarm stack from the full featured docker based Plone Volto deployment 
+-   Created by current Cookieplone project template including backend, frontend, postgres, traefik, varnish:
+
+#### Create a new Zope Manager User with a non existing user-ID
+
+-   Enter the host via ssh as root
+-   list the running containers: docker ps
+-   enter a shell in the first backend container listed: `docker exec -it [[4-digitPartOfID]] bash`
+-   run command in the app folder: `./docker-entrypoint.sh bin/addzopeuser -c /app/etc/relstorage.conf userid password`
```

Comment:
The conf file can also be zope.conf or zeo.conf depending on the configuration. I submitted https://github.com/plone/plone-backend/pull/175 which will make it so you can just run `./docker-entrypoint.sh addzopeuser userid password` -- but that won't be available until it's merged and the next Plone 6.1.x image is released.